engine/apps/webhooks/tasks/trigger_webhook.py

```diff
@@ -119,12 +121,16 @@ def make_request(webhook, alert_group, data):
                 status["request_data"] = request_kwargs.get("data")
             response = webhook.make_request(status["url"], request_kwargs)
             status["status_code"] = response.status_code
-            try:
-                status["content"] = json.dumps(response.json())
-            except JSONDecodeError:
-                status["content"] = response.content.decode("utf-8")
+            content_length = response.headers.get("Content-Length")
+            if content_length and int(content_length) < WEBHOOK_RESPONSE_LIMIT:
+                try:
+                    status["content"] = json.dumps(response.json())
+                except JSONDecodeError:
+                    status["content"] = response.content.decode("utf-8")
+            else:
+                status["content"] = f"Response content exceeds {WEBHOOK_RESPONSE_LIMIT} character limit"
         else:
```

Comment:
Wondering if we could remove this else block, and update the closing return to be:
`return triggered, status, error, exception`